Editor/UI/ProjectAuditorAnalytics.cs

```diff
@@ -0,0 +1,401 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.Analytics;
+
+namespace Unity.ProjectAuditor.Editor.UI
+{
+    internal class ProjectAuditorAnalytics
+    {
+        const int k_MaxEventsPerHour = 100;
+        const int k_MaxEventItems = 1000;
+        const int k_MaxIssuesInAnalyzeSummary = 10;
+
+        const string k_VendorKey = "unity.projectauditor";
+        const string k_EventTopicName = "projectAuditorUsage";
+
+        static bool s_EnableAnalytics = false;
+
+        public static void EnableAnalytics()
+        {
+#if UNITY_2018_1_OR_NEWER
+            AnalyticsResult result = EditorAnalytics.RegisterEventWithLimit(k_EventTopicName, k_MaxEventsPerHour, k_MaxEventItems, k_VendorKey);
+            if (result == AnalyticsResult.Ok)
+                s_EnableAnalytics = true;
+#endif
+        }
+
+        public enum UIButton
+        {
+            Analyze,
+            Export,
+            ApiCalls,
```

Comment:
Since we might add new categories/tabs (like Assets), could we move ApiCalls and ProjectSettings to the bottom and maybe assign 100 to ApiCalls so they are in their own range?